packages/main/src/YearPicker.js

```diff
@@ -65,8 +66,8 @@ const metadata = {
 	},
 };
 
-const PAGE_SIZE = 20; // Total years on a single page
-const ROW_SIZE = 4; // Years per row (5 rows of 4 years each)
+// const PAGE_SIZE = 20; // Total years on a single page
+// const ROW_SIZE = 4; // Years per row (5 rows of 4 years each)
```

Comment:
Why are these two lines commented? Probably you forgot to delete them?
There are new constants defined below... But why there are separate constanst with same name in several methods?

Reply:
yes, I forgot for this.
Done